Makefile

```diff
@@ -27,13 +27,5 @@ clean-test:
 	rm -f .coverage
 	rm -fr htmlcov/
 
-test:
-	python setup.py test --pytest-args="--cov=./djmoney tests"
```

Comment:
You're right, I'll probably start only with tox.ini update :) Other things stay be untouched - I don't know how to solve duplicated dependencies problem yet, but anyway it will not make things worse - tox will have a complete list of dependencies without possible conflicts with setup.py.